portfolio/src/main/webapp/header.txt

```diff
@@ -0,0 +1 @@
+document.write('<div id="header"><p class="header-text"> Shuli Jones </p><div id="navbar"><a href="index.html" class="navlink">Home</a> <div class="dropdown"><a href="projects.html" class="navlink">Projects</a><div class="dropdown-list"><a href="projects.html#task-manager">Task Manager</a><a href="projects.html#matching-alg">Matching Algorithm </a><a href="projects.html#loft">Loft Bed</a></div></div> <div class="dropdown"><a href="elsewhere.html" class="navlink">Elsewhere</a><div class="dropdown-list"><a href="https://github.com/shulijones">GitHub</a><a href="https://mitadmissions.org/blogs/author/jonsh/">MIT Admissions</a></div></div></div></div>')
```

Comment:
Another option is to move this to the main javascript file and use something like <body onload="setHeader()"> and the setHeader function will add html to a specific div element identified by id (<div id="header"></div>)

Reply:
Thank you for all of your feedback on this! I'll shift to a multi-line string for now, and will look into using a setHeader javascript function.